docs/datamodel/links.rst

```diff
@@ -475,11 +475,141 @@ in these cases due to better ergonomics of selecting, updating, and even
 casting into :eql:type:`json` when keeping all data in the same place rather
 than spreading it across link and object properties.
 
+
+Inserting and updating link properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To add a link with a link property, add the link property to a shape on the
+linked object being added. Be sure to prepend the link property's name with
+``@``.
+
+.. code-block:: edgeql
+
+    insert Person {
+      name := "Bob",
+      family_members := (
+        select detached Person {
+          @relationship := "sister"
+        }
+        filter .name = "Alice"
+      )
+    }
+
+The shape could alternatively be included on an insert if the object being
+linked (the ``Person`` named "Alice" in this example) is being inserted as part
+of the query. If the outer person ("Bob" in the example) already exists and
+only the links need to be added, this can be done in an ``update`` query
+instead of an ``insert`` as shown in the example above.
+
+Updating a link's link property is similar to adding a new one except you can
```

Comment:
This is good to point out and I think could be talked up a bit more to ensure that the reader notices that it's a bit convenient, something like this perhaps:

"Updating a link's property  is similar to adding a new one except that now you don't need to select from the object type: you can select the link on the object itself because the link has already been established."